src/Element/NoteReaction.tsx

```diff
@@ -53,8 +55,9 @@ export default function NoteReaction(props: NoteReactionProps) {
         showHeader: ev?.Kind === EventKind.Repost,
         showFooter: false,
     };
+    const isOpMuted = root && isMuted(root.pubkey)
 
-    return (
+    return isOpMuted ? null : (
```

Comment:
For notes I added the `HiddenNote`, do you think is worth adding for reactions too? thought we could just hide them since they can get spammy

Reply:
We dont show pure reactions currently, only reposts, im thinking of reposts where the person being reposted in muted, if the reposter is muted then you wont see it anyway because it will be removed at timeline/thread level